kedro/framework/cli/starters.py

```diff
@@ -95,7 +97,44 @@ class KedroStarterSpec:
 KEDRO_PATH = Path(kedro.__file__).parent
 TEMPLATE_PATH = KEDRO_PATH / "templates" / "project"
 
-_STARTERS_REPO = "git+https://github.com/kedro-org/kedro-starters.git"
+
+def _get_latest_starters_version() -> str:
+    if "STARTERS_VERSION" not in os.environ:
```

Comment:
```suggestion
    if "KEDRO_STARTERS_VERSION" not in os.environ:
```

? (and everywhere else)